client/checkout/platform-checkout/style.scss

```diff
@@ -45,6 +45,17 @@
 	}
 }
 
+.disabled-page {
```

Comment:
This is not part of any approved design. I added this as a solution to prevent the user from doing anything on the checkout page while the request for checking a platform user session is going on.